spec/controllers/faye_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe FayeController do
+  include Devise::TestHelpers
+  before :each do
+    user = create :user, admin: true
```

Comment:
Apologies, there were times in tests where we needed to have a variable for a user that is admin.